sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -107,7 +107,65 @@ APIs for managing configurations for devices and modules.
 <details>
 
 ```csharp
+/// <summary>
+/// Gets a configuration on the IoT Hub for automatic device/module management
+/// </summary>
+/// <param name="configurationId">The unique identifier of the configuration.</param>
+/// <param name="cancellationToken">The cancellation token.</param>
+/// <returns>TwinConfiguration for a automatic device/module management</returns>
+public virtual async Task<Response<TwinConfiguration>> GetConfigurationAsync(string configurationId, CancellationToken cancellationToken = default)
+
+/// <summary>
+/// Creates a configuration on the IoT Hub for automatic device/module management
+/// </summary>
+/// <param name="configuration">Twin configuration to create</param>
+/// <param name="cancellationToken">The cancellation token.</param>
+/// <returns>TwinConfiguration for a automatic device/module management</returns>
```

Comment:
Same comment as above. If the method creates an "automatic device/module management configuration", then it should return a "automatic device/module management configuration"

Reply:
Restful APIs usually follow the pattern like GET device identity, CREATE device identity, DELETE  device identity, so you need to figure out what noun replaces "device identity" above and use that for the comments here

Reply:
sure

Reply:
I still don't see you following a restful pattern with the description of the return type here. "Returns the created automatic device/module management twin configuration"?